### ISSUE TITLE
PostgreSQL warns about data directory structure for storing database files

### ISSUE THREAD

**Issue by khashf** [hackoregon/civic-devops#28]
In the fundamentally same environment & instance format, I see the following warning from PostgreSQL while the `create-database-development.sh` script is running:

```
Creating properly-configured $PGDATA data_directory...
Configuring override.conf to use the non-default data_directory...

[Service]
Environment=PGDATA=/data/databases
Initializing PostgreSQL...
WARNING: Note that either your data directory '/data/databases' or
         the parent directory '/data'
         is a direct mountpoint.  This is usually a bad idea and your
         filesystem layout should ideally look like:
         /ROOT_OWNED_MOUNTPOINT/POSTGRES_OWNED_DIRECTORY/**DATADIR**.
         See the upstream documentation for more info:
         http://www.postgresql.org/docs/9.6/static/creating-cluster.html
 * Initializing database in '/data/databases'
 * Initialized, logs are in /var/lib/pgsql/initdb_postgresql.log

```
We already have:
-  /data as mount point
- /data/databases as PostgreSQL owned directory

Should we add another directory inside /data/databases, says `postgresql` as the mentioned **DATADIR** to contain PostgreSQL files instead of just /data/database?

**Comment by khashf** [comment]
I side with PostgreSQL advice. Having one more directory depth doesn't hurt. Once @MikeTheCanuck confirms to fix this, I'll do.

**Comment by MikeTheCanuck** [comment]
Yes, please change the data directory to `/data/postgres/databases`.

**Comment by MikeTheCanuck** [comment]
Fixed in PR #33
